### PR TITLE
Improvement/update vulnerable dependencies

### PR DESCRIPTION
**Pull request by jwtlake**
# Asana
- https://app.asana.com/0/162113178729864/532116282868842

# Problem
- github flagged some potential security vulnerabilities in our dependencies

# Solution
- bump jquery dep
- lint fixes
- added npm package lock file
- commit updated dist files

# This Package is used by
https://github.com/MRN-Code/coins/blob/develop/package.json#L21
https://github.com/MRN-Code/coins_core/blob/develop/js/browserApp/package.json#L35

# Pre Publish Tests
```
-- manually install in projects
npm install git://github.com:MRN-Code/coins-logon-widget.git#improvement/update-vulnerable-dependencies --save

-- manually trigger timeout popup (from chrome console)
coinsUtils.clientAuthTimer.showLoginPopup();

-- restart webpack server
sudo systemctl restart webpack-dev-server

```

# Post Publish Tests
Revert any manual github npm installs
Bump `coins-logon-widget` version in `coins_core` and `coins`
Re-test

### CODE REVIEW

File: dist/coins-logon-widget.css [hunk 1]
```diff
@@ -335,25 +323,21 @@ input[type="text"].coins-logon-widget-input {
  * button display horizontally.
  */
 .coins-logon-widget-form-horizontal {
-  -webkit-align-items: flex-start;
-      -ms-flex-align: start;
-          align-items: flex-start;
-  display: -webkit-flex;
+  -ms-flex-align: start;
```
Comment (mstone121) [inline dist/coins-logon-widget.css:326]: Why did you remove the webkit stuff? Shouldn't we keep it for cross browser support?

Comment (jwtlake) [reply dist/coins-logon-widget.css:326]: It wasn't intentional. I just committed the dist output generated from the `npm run build`

**Review by mstone121 — APPROVED** [review]
